Vienna/Sources/Main window/ArticleListView.m

```diff
@@ -1287,8 +1292,8 @@ -(id)tableView:(NSTableView *)aTableView objectValueForTableColumn:(NSTableColum
 			[summaryString appendFormat:@"%@", folder.name];
 			delimiter = @" - ";
 		}
-		if ([db fieldByName:MA_Field_Date].visible) {
-			[summaryString appendFormat:@"%@%@", delimiter, [NSDateFormatter vna_relativeDateStringFromDate:theArticle.date]];
+		if ([db fieldByName:MA_Field_LastUpdate].visible) {
+			[summaryString appendFormat:@"%@%@", delimiter, [NSDateFormatter vna_relativeDateStringFromDate:theArticle.lastUpdate]];
 			delimiter = @" - ";
 		}
```

Comment:
Should we add the publication date here, too?

Reply:
No, as we have not added the "Date Published" field to the vertical layout.
All this part of the code is subject to the condition  that we are generating an headline for vertical layout:
https://github.com/ViennaRSS/vienna-rss/blob/d692efbe39e305202e588d39cd313673542ebc62/Vienna/Sources/Main%20window/ArticleListView.m#L1239


The same omission apply to MA_Field_Enclosure (enclosure link)

Reply:
I think there should be an option which date to use, maybe even per feed; we can add that later though.